lib/view_model/active_record/cache.rb

```diff
@@ -10,32 +10,22 @@ class UncacheableViewModelError < RuntimeError; end
 
   attr_reader :viewmodel_class
 
-  def initialize(viewmodel_class)
+  # If model_cache_group: is specified, it must be a group of a single key: `:id`
+  def initialize(viewmodel_class, model_cache_group: nil)
```

Comment:
Since we're making the parent cache generic and application owned, let's call it `parent_cache_group`.